src/powershell/Public/ConvertTo-FinOpsSchema/ConvertTo-FinOpsSchema.ps1

```diff
@@ -0,0 +1,221 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+
+<#
+.SYNOPSIS
+Converts Cost Management cost data to the FinOps Open Cost and Usage Specification (FOCUS) schema.
+
+.DESCRIPTION
+The ConvertTo-FinOpsSchema command returns an object that adheres to the FinOps Open Cost and Usage Specification (FOCUS) schema.
+It currently understands how to convert Cost Management cost data using the latest schemas as of September 2023.
+Older schemas may not be fully supported. Please review output and report any issues to https://aka.ms/ftk.
+You can pipe objects to ConvertTo-FinOpsSchema from an exported or downloaded CSV file using Import-Csv or ConvertFrom-Csv and pipe to Export-Csv to save as a CSV file.
+
+.PARAMETER ActualCost
+Specifies the actual cost data to be converted. Object must be a supported Microsoft Cost Management schema.
+
+.PARAMETER AmortizedCost
+Specifies the amortized cost data to be converted. Object must be a supported Microsoft Cost Management schema.
+
+.PARAMETER Version
+FOCUS schema version to convert data to. Allowed values: 0.5, 1.0-workingdraft, 1.0-preview. Default = 0.5.
+
+.EXAMPLE
+ConvertTo-FinOpsSchema -ActualCost (Import-Csv my-actual-cost-details.csv) -AmortizedCost (Import-Csv my-amortized-cost-details.csv) | Export-Csv my-cost-details-in-focus.csv
+
+.NOTES
+Version: 
+Author: 
+Date: 
```

Comment:
A few comments:
1. Tab these in
2. nit: Should we just use "FOCUS" in the synopsis to make it shorter? 🤔
3. Should we leave out the Version parameter until we bring in other versions? That way, we could ship one version with just 0.5, then follow up with the others shortly after. (I'm suggesting this so we can get this into the next FTK release 🙂) 
4. We don't use the file to track versions. That's all in the repo. I'd leave that part out.
```suggestion
    .SYNOPSIS
    Converts Cost Management cost data to the FOCUS schema.
    
    .DESCRIPTION
    The ConvertTo-FinOpsSchema command returns an object that adheres to the FinOps Open Cost and Usage Specification (FOCUS) schema.
    
    This command supports Cost Management cost data using the latest schemas as of September 2023.
    Older schemas may not be fully supported. Please review output and report any issues to https://aka.ms/ftk.
    You can pipe objects to ConvertTo-FinOpsSchema from an exported or downloaded CSV file using Import-Csv or ConvertFrom-Csv and pipe to Export-Csv to save as a CSV file.
    
    .PARAMETER ActualCost
    Specifies the actual cost data to be converted. Object must be a supported Microsoft Cost Management schema.
    
    .PARAMETER AmortizedCost
    Specifies the amortized cost data to be converted. Object must be a supported Microsoft Cost Management schema.
    
    .PARAMETER Version
    FOCUS schema version to convert data to. Allowed values: 0.5, 1.0-workingdraft, 1.0-preview. Default = 0.5.
    
    .EXAMPLE
    ConvertTo-FinOpsSchema -ActualCost (Import-Csv my-actual-cost-details.csv) -AmortizedCost (Import-Csv my-amortized-cost-details.csv) | Export-Csv my-cost-details-in-focus.csv
    
    .LINK
    https://aka.ms/ftk/ConvertTo-FinOpsSchema
```